controller/controller_server/messages.py

```diff
@@ -3,7 +3,12 @@
 ObjectIdError_message = "Wrong {0} id format : {1}"
 
 # validation error messages
-invalid_secret_message = "invalid secret was passed"
+invalid_secrets_config_message = "invalid secrets config was passed"
```

Comment:
secret*?

Reply:
done

Reply:
same for `secrets_missing_topologies_message` var name

Reply:
it is changed in the mesasage

Reply:
I know, which is good, but I think the var name should be aligned accordingly

Reply:
done

Reply:
I still see `invalid_secrets_config_message` (plural)

Reply:
wired, I can't.. can you link the occurrences?

Reply:
now I can't see it either, must have been on my side. resolving